src/github.com/stellar/federation/record.go

```diff
@@ -1,8 +1,12 @@
 package federation
 
-type Record struct {
-	Username  string `db:"username" json:"stellar_address"`
-	AccountId string `db:"account_id" json:"account_id"`
-	MemoType  string `db:"memo_type" json:"memo_type,omitempty"`
```

Comment:
You've removed `omitempty` and when memo fields are empty response looks like this:
```json
{"stellar_address":"bob*stellar.org","account_id":"GD3JZSKTEVO74DJMDV6CMUJGOEIS67KVKH3A7LWVLAJ4F5EG4XUO7MBY","memo_type":"","memo":""}
```

Also, imho we don't need two structures since responses are the same for federation and reverse federation. Previous version:
```go
type Record struct {
	Username  string `db:"username" json:"stellar_address"`
	AccountId string `db:"account_id" json:"account_id"`
	MemoType  string `db:"memo_type" json:"memo_type,omitempty"`
	Memo      string `db:"memo" json:"memo,omitempty"`
}
```
seems fine to me.